ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -14,6 +14,8 @@ case class Config[Extra](
     address: Option[String],
     port: Int,
     portFile: Option[Path],
+    serverJdbcUrl: String,
```

Comment:
This now requires a SQL backend as the kvutil ledger. Could the Runner then still be used with another implementation? Maybe `Runner` is not supposed to be used that way, then it's fine.

Reply:
It defaults to H2 in memory, just as it did before; this is how _ledger-on-memory_ works.

Reply:
Right. I think it's not clear to me what the target of the `kvutils/app` folder is. On the surface it seems very general and not tied to a specific `ledger-on-...` implementation.

Reply:
It is, absolutely. But it is tied to both _kvutils_, the Ledger API Server, and the Indexer; for the API Server/Indexer, it'll use H2 by default.